integration_tests/dbt_project.yml

```diff
@@ -32,12 +32,14 @@ vars:
     greenhouse_scorecard_attribute_identifier: "scorecard_attribute"
     greenhouse_social_media_address_identifier: "social_media_address"
     greenhouse_source_identifier: "source"
-    greenhouse_tag_identifier: "tag"
-    greenhouse_user_identifier: "user"
+    # greenhouse_tag_identifier: "tag"
+    # greenhouse_user_identifier: "user"
+    greenhouse_tags_identifier: "tags"
+    greenhouse_users_identifier: "users"
     greenhouse_user_email_identifier: "user_email"
     scheduled_interview: "{{ fivetran_utils.seed_data_helper('scheduled_interview', ('snowflake', 'postgres')) }}"
-  greenhouse_application_custom_columns: ['custom_what_is_your_preferred_pronoun_', 'custom_how_did_you_hear_about_fivetran_']
-  greenhouse_job_custom_columns: ['custom_employment_type', 'custom_stock_options']
+  # greenhouse_application_custom_columns: ['custom_what_is_your_preferred_pronoun_', 'custom_how_did_you_hear_about_fivetran_']
+  # greenhouse_job_custom_columns: ['custom_employment_type', 'custom_stock_options']
```

Comment:
Were these intentionally commented out?

Reply:
@fivetran-joemarkiewicz  yes since I removed the `custom_` columns from the seed data. I kept them in the project yml for future testing, but come to think of it, if it's gone from the seed data, I could just remove it here as well.